test/unit/uk/gov/hmrc/exports/connectors/CustomsDeclarationsConnectorSpec.scala

```diff
@@ -45,13 +45,12 @@ class CustomsDeclarationsConnectorSpec extends UnitSpec with MockitoSugar with M
     val conversationId = "123456"
     val responseHeaders: Map[String, Seq[String]] = Map("X-Conversation-ID" -> Seq(conversationId))
 
-    val mockHttpResponse: CustomsDeclarationsResponse =
-      CustomsDeclarationsResponse(Status.ACCEPTED, Some(testConversationId))
+    val mockHttpResponse = Future.successful(CustomsDeclarationsResponse(Status.ACCEPTED, Some(testConversationId)))
```

Comment:
What is the purpose of wrapping the `Response` in Future?

Reply:
Because `httpClient.POSTString` returns a `Future[T]`